src/app/modules/shared/components/details-fields/details-fields.component.spec.ts

```diff
@@ -311,7 +315,7 @@ describe('DetailsFieldsComponent', () => {
     spyOn(toastrServiceStub, 'error');
 
     const futureDate = moment().add(1, 'days').format('YYYY-MM-DD');
-    component.entryForm.setValue({ ...formValues, entry_date: futureDate });
+    component.entryForm.setValue({ ...formValues, entry_date: futureDate, departure_date: futureDate });
     component.onSubmit();
```

Comment:
You need to add a couple of tests here:
1. start_date in the future / end date OK
2. start_date OK / end date in the future

Reply:
@enriquezrene done